ydb/docs/en/core/integrations/migration/_includes/toc-table.md

```diff
@@ -0,0 +1,8 @@
+# Data Migrations
```

Comment:
Currently, most of the headers in the documentation capitalize only the first letter; let's keep it consistent.

I'd personally also prefer to switch over to some variation of headline case https://en.wikipedia.org/wiki/Title_case at some point, but let's discuss and potentially do this separately.